server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/controllers/EarthquakeController.java

```diff
@@ -133,6 +139,21 @@ public Earthquake createEarthquake(
         // TODO finish adding log statements
         // First, get the Earthquake object from the form
         // TODO what should be done if a user sends multiple earthquake objects?
+
+        // check if the user's number of the hazard is within the allocation
+        boolean postOk = AllocationUtils.checkNumHazard(allocationRepository, spaceRepository, this.username);
+        if (postOk == false) {
```

Comment:
Since postOk is not used anywhere else, you can combine this into a single line for better readability:

if(!AllocationUtils.checkNumHazard(allocationRepository, spaceRepository, this.username)) {...}

Reply:
Changed. Also changed the same thing in other hazard controllers.